dts/msm8226/msm8926-v1-mtp.dts

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	compatible = "qcom,msm8926-mtp", "qcom,msm8926";
+	qcom,msm-id = <200 0>;
+	qcom,board-id = <8 0>;
+
+	/* All the Nokia devices are expected to use lk1st, which passes the 
```

Comment:
I guess a small nit would be to say that those nodes aren't really sorted but don't think we really kept them sorted in other files as well...